bots/hermes/makefile

```diff
@@ -0,0 +1,42 @@
+.DEFAULT_GOAL=help
```

Comment:
Can you rename this `bots/hermes/makefile` file with a Uppercase `M` like `bots/hermes/Makefile` ?